accesspoc/accesspoc/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+This will make sure the Celry app is always imported
+when Django starts so that shared_task will use it.
+"""
```

Comment:
Should this be a docstring? http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html